lang/hr/index.md

```diff
@@ -0,0 +1,262 @@
+
+Najjednostavniji je način da započnemo inicijalni razvoj izdavanjem verzije
+0.1.0 te povećavamo oznaku manje verzije za svako sljedeće izdanje.
+
+### Kako znamo kada izdati verziju 1.0.0?
+
+Ako se softver koristi u produkciji, već bi vjerojatno trebao biti na verziji
+1.0.0. Ako već imamo stabilni API, na koji korisnici mogu računati, trebao bi
+biti na verziji 1.0.0. Ako smo zabrinuti zbog kompatibilnosti unatrag, softver
+bi već trebao biti izdan pod verzijom 1.0.0.
+
+### Ne sprječava li ovo ubrzani razvoj i brzo povećavanje verzija?
+
+Veća verzija nula se svodi na brzi razvoj. Ako mijenjamo API svaki dan, trebali
+bi ostati na verziji 0.y.z ili na posebnoj grani raditi na sljedećoj većoj
+verziji.
+
+### Ako i najmanje unatrag inkompatibilne promjene u APIju zahtijevaju povećanje veće verzije, nećemo li vrlo brzo doći do verzije 42.0.0?
+
+Ovo je pitanje odgovornog razvoja i predviđanja. U softver koji ima puno
+ovisnog koda, inkompatibilne promjene potrebno je uvoditi postepeno. Cijena
+nadogradnje može biti značajna. Ako moramo povećati veću verziju, kako bismo
+izdali verziju s inkompatibilnim promjenama, moramo razmisliti o posljedicama
+tih promjenama i procijeniti odnos cijene i koristi.
+
+### Dokumentacija čitavog javnog APIja zahtijeva previše posla!
+
+Naša je odgovornost kao profesionalnih programera da ispravno dokumentiramo
+softver koji je namijenjen korisnicima. Upravljanje složenosti softvera vrlo
+je važno kako bismo projekt održali efikasnim, što je teško ako korisnici ne
+znaju kako koristiti naš softver ili koje metode mogu zvati. Semantičko
+verzioniranje i kvalitetno definirani Semantički verzionirani javni API osigurat
+će da sve funkcionira kako treba.
+
+### Što ako slučajno izdamo unatrag inkompatibilne promjene kao manju verziju?
+
+Čim primijetimo da smo prekršili specifikaciju Semantičkog verzioniranja,
+potrebno je ispraviti pogrešku te izdati manju verziju koja će ispraviti problem
+i vratiti kompatibilnost. Čak i u takvim uvjetima, nije prihvatljivo mijenjati
+verzionirana izdanja. Ako je prikladno, dokumentiramo verziju koja krši
+specifikaciju te informiramo korisnike kako bi je bili svjesni.
+
+### Što činiti ako promijenimo ovisnosti bez mijenjanja javnog APIja?
+
+Takve promjene smatramo kompatibilnima jer ne utječu na javni API. Softver koji
+eksplicitno ovisi o istim ovisnostima kao i naš paket treba imati vlastite
+specifikacije ovisnosti, a autor će se brinuti o konfliktima. Je li promjena
+na razini zakrpe ili manje verzije, ovisi o tome jesmo li dodali ovisnosti
+kako bi ispravili bug ili dodali nove funkcionalnosti. U posljednjem slučaju
+možemo očekivati i dodatni kod, pri čemu se očito radi o novoj manjoj verziji.
+
+### Što ako nehotice promijenimo javni API na način koji ne odgovara promjeni oznake verzije (npr. u kodu neispravno uvodimo veću unatrag inkompatibilnu promjenu unutar izdanja zakrpe)?
+
+Pokušamo najbolje prosuditi sami. Ako imamo velik broj korisnika, na koje će
```

Comment:
"Ako imamo" -> "Ako imate"

Reply:
I might be more than a little inconsistent here - parts are written in the second person, other in the first person plural.